src/comment-pr/run.ts

```diff
@@ -16,6 +16,8 @@ export async function run(): Promise<void> {
 
   const octokit = github.getOctokit(githubToken);
 
+  const stage = process.env.STAGE ?? 'stg';
+  console.log(stage);
```

Comment:
I'm fine with this, but just checking if you wanted to keep the log!

Reply:
Thanks for pointing this out! Will remove.